### ISSUE TITLE
Label is announced twice for role="checkbox" instances

### ISSUE THREAD

**Issue by peasandwell** [nvaccess/nvda#7948]
### Steps to reproduce:
1. Open IE11
2. Browse to https://codepen.io/peasandwell/pen/oEXorY
3. Focus onto custom checkbox
4. Notice label is announced twice

### Expected behavior:
Label should be announced once, as it would in Firefox.

### Actual behavior:
The label for the custom checkbox is announced twice (i.e. Lorem ipsum Lorem ipsum checkbox not checked")

### System configuration:
NVDA version: 2017.4 on Windows 10, tested on IE11.

**Comment by feerrenrut** [comment]
I can reproduce this with the provided codepen I got the following results:
- IE11 speaks twice
- Chrome speaks twice
- Firefox speaks once
- Edge speaks once

**Comment by feerrenrut** [comment]
Unlike #7960 this issue also affects 2017.3

**Comment by nhaidaraly** [comment]
The issue is still present on IE11 but was solved on Chrome, Firefox and Edge...

**Comment by DiegoBM** [comment]
I can reproduce this issue in Chrome consistently

**Comment by DiegoBM** [comment]
It also happens for native check-boxes by the way, not just custom ones.

This issue is dating back January last year and nobody has worked on it on more than one year, is it something that was agreed to stay as it is?

**Comment by Adriani90** [comment]
cc: @leonardder, @jcsteh

**Comment by Adriani90** [comment]
@feerrenrut could this also be a duplicate of #9155?

**Comment by Adriani90** [comment]
Closing as fixed, in Chrome and Edge 112 as well as Firefox 111 when focusing the checkbox either in focus or in browse mode, NVDA speaks the checkbox only once.
If you still have this issue, please comment with updated steps to reproduce and we can reopen.
Note that support for IE ended in 2022 completely and it is automatically removed from machines that still have it after an Edge update in February 2023.
